Makefile

```diff
@@ -140,6 +141,11 @@ ol: OL_JS = ol.js ol-debug.js
 ol: scripts/ol-geoadmin.json .build-artefacts/ol3 .build-artefacts/ol-requirements-installation.timestamp
 	cd .build-artefacts/ol3; \
 	git reset HEAD --hard; \
+	ifdef ${OL3_VERSION} \
+	git checkout ${OL3_VERSION}; \
+	else \
+	git checkout $(OL3_VERSION); \
+	enif; \
```

Comment:
not the same:
` ${OL3_VERSION}` refers to an argument 
`$(OL3_VERSION)` refers to a variable in makefile